inc/cleantalk-ajax.php

```diff
@@ -667,6 +668,7 @@ function ct_ajax_hook($message_obj = null)
     if ( apbct_is_exception_arg_request() ) {
         $base_call_params['exception_action'] = 1;
         $base_call_params['sender_info']['exception_description'] = apbct_is_exception_arg_request();
+        $base_call_params['sender_info']['$sender_emails_array'] = $sender_emails_array;
```

Comment:
`$base_call_params['sender_info']['$sender_emails_array']` -> `$base_call_params['sender_info']['sender_emails_array']`
Typo is in the array key name `sender_emails_array`

Reply:
ok